app/celery/tasks.py

```diff
@@ -81,6 +81,15 @@
 from app.utils import get_csv_max_rows
 
 
+@notify_celery.task(name="process-inflight")
+@statsd(namespace="tasks")
+def process_inflight(receipt, results, type):
+    if type == SMS_TYPE:
+        save_smss.apply_async((None, results, receipt), queue=QueueNames.DATABASE)
```

Comment:
In order to not overflow the `DATABASE` queue even further with more messages, we might want to use a different queue such as `QueueNames.NOTIFY_CACHE` instead. We already overflow the `DATABASE` queue at the moment as it receives all notification to save down the pipeline.